test/spec/todo-item/todo-item.spec.ts

```diff
@@ -91,3 +91,34 @@ describe('POST /todos', () => {
     }
   })
 })
+
+describe('GET /todos/:id', () => {
```

Comment:
We should also have a check for whether API returns the Todo with matching database ID

Reply:
I couldn't get the last comment.